src/project-conf/index.ts

```
@@ -311,200 +692,1082 @@ function resolveConfigPaths(
+  param: string,
+  preset: ConfigurePreset
+): any {
+  switch (param) {
+    case "idf.cmakeCompilerArgs":
+      return getESPIDFSettingValue(preset, "compileArgs") || "";
+    case "idf.ninjaArgs":
+      return getESPIDFSettingValue(preset, "ninjaArgs") || "";
+    case "idf.buildPath":
+      return preset.binaryDir || "";
+    case "idf.sdkconfigDefaults":
+      const sdkconfigDefaults = preset.cacheVariables?.SDKCONFIG_DEFAULTS;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)